app/scripts/components/common/smart-link.tsx

```diff
@@ -12,8 +12,25 @@ export default function SmartLink(props: SmartLinkProps) {
   const { to, ...rest } = props;
 
   return /^https?:\/\//.test(to) ? (
-    <a href={to} {...rest} />
+    <a target='_blank' rel='noopener noreferrer' href={to} {...rest} />
   ) : (
     <Link to={to} {...rest} />
   );
 }
+
+interface CustomLinkProps {
+  href: string;
+}
+
+/**
+ * For links defined as markdown, this component will open the external link in a new tab.
+ */
+export function CustomLink(props: CustomLinkProps) {
+  const { href, ...rest } = props;
+
+  return /^https?:\/\//.test(href) ? (
+    <a target='_blank' rel='noopener noreferrer' href={href} {...rest} />
+  ) : (
+    <Link to={href} {...rest} />
+  );
```

Comment:
This gives us no other options but to open external links in a new tab. Not sure if we want this behavior though... This should probably be decided on the instance level?